config.nims

```diff
@@ -1,5 +1,3 @@
-# Nim compilation defaults for the whole repository
-#--gc: arc
```

Comment:
If the --gc: arc was commented, that is a merging error, please uncomment it directly in this file

Reply:
See above